confgenerator/testdata/valid/linux/logging-receiver_kafka/input.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logging:
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration, enable_<APP> files and the main `confgenerator/testdata/valid/{LINUX_OR_WINDOWS}/{logging_or_metrics}-receiver_{APP}/input.yaml` test case, because some users look at the `valid` cases for sample configurations as well. Additional more complex test cases could use more complex receiver or pipeline ID. 

See more context in https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764

Action items:
* Make `logging-receiver_kafka/input.yaml` test the bare minimum config
* Move the custom test case to a separate test case (It could be combined with `logging-receiver_kafka_refresh_interval` and get renamed to `logging-receiver_kafka_custom`).